<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2Pal - Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QTextEdit" name="chatLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="messageLayout">
      <item>
       <widget class="QLineEdit" name="inputField">
        <property name="placeholderText">
         <string>Type your message...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="peerLayout">
      <item>
       <widget class="QListWidget" name="peerList">
        <property name="toolTip">
         <string>Select a peer to chat privately.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="peerButtons">
        <item>
         <widget class="QPushButton" name="privateChatButton">
          <property name="text">
           <string>Private Chat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPeerButton">
          <property name="text">
           <string>Add Peer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="privateMessageLayout">
      <item>
       <widget class="QLineEdit" name="privateInputField">
        <property name="placeholderText">
         <string>Type private message...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendPrivateMessageButton">
        <property name="text">
         <string>Send Private</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>Search files...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="searchResultsTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column>
       <property name="text">
        <string>Filename</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size (KB)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Source Node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Download</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Progress</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>

